Tests/Unit/RealRemoteConfigTests.swift

```diff
@@ -1,3 +1,45 @@
 import Quick
 import Nimble
 @testable import RRemoteConfig
+
+class RealRemoteConfigSpec: QuickSpec {
+    class CacheMock: ConfigCache {
+        var getStringCalled = false
+        var refreshCalled = false
+
+        override func refreshFromRemote() {
+            self.refreshCalled = true
+        }
+
+        override func getString(_ key: String, _ fallback: String) -> String {
+            self.getStringCalled = true
+            return fallback
+        }
+    }
+    override func spec() {
+        describe("getString function") {
+            it("calls config cache get string") {
+                RealRemoteConfig.shared.cache = CacheMock(fetcher: ConfigFetcher(client: APIClient(), environment: Environment()), poller: Poller())
+
+                _ = RealRemoteConfig.shared.getString("foo", "bar")
+
+                expect((RealRemoteConfig.shared.cache as? CacheMock)?.getStringCalled).to(equal(true))
```

Comment:
You could change the `getStringCalled` variable to something like `stringReturnValue`. And then in this test you can do something like:
```
RealRemoteConfig.shared.cache.stringReturnValue = "some_value"
expect(RealRemoteConfig.shared.getString("foo", "bar")).to(equal("some_value"))
```
Will that work?